packages/scaffolding-cli/templates/src/csr/config/env.js

```diff
@@ -89,8 +88,10 @@ function getClientEnvironment(publicUrl) {
                 WDS_SOCKET_PATH: process.env.WDS_SOCKET_PATH,
                 WDS_SOCKET_PORT: process.env.WDS_SOCKET_PORT,
                 // Stacks: adding in external api URL
-                MENU_API_URL: process.env.MENU_API_URL,
-                APPINSIGHTS_KEY: process.env.APPINSIGHTS_KEY || "",
+                MENU_API_URL:
+                    process.env.MENU_API_URL || "http://localhost:3000",
```

Comment:
Remove the port from the URL

Reply:
In the cypress setup, I followed the runtime script to resolve the env config:

```js
    // Get the current NODE_ENV and determine the correct app paths from it
    const env = getClientEnvironment(paths.publicUrlOrPath.slice(0, -1))
    const protocol = process.env.HTTPS === "true" ? "https" : "http"
    const port = process.env.PORT || 3000
    const HOST = process.env.HOST || "0.0.0.0"
    const baseUrl = env.raw.PUBLIC_URL || HOST

    config.baseUrl = `${protocol}://${baseUrl}:${port}/${env.raw.APP_BASE_PATH}`
```

Do we need to change this approach?

Reply:
no that's fine for the above, but for these cases we should leave it as is because it's either going to be a standard PORT like 443 (which we don't need to specify) or totally without port in case of SSR using a K8s `<service>.<namespace>` notation